source/_drafts/Introducing-Thumbcoil.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Introducing Thumbcoil
+author:
+  name: Jon-Carlos Rivera
+  github: imbcmdth
+tags:
+---
+
+{% pullquote right %}
+A **transmuxer** takes media contained in some file format, extracts the raw compressed video and audio from inside (a process called demuxing) and repackages the compressed data  into another format (termed remuxing) without performing any re-compression.
+{% endpullquote %}
+
+## In the Beginning
+
+While building [Mux.js](https://github.com/videojs/mux.js) - the transmuxer at the heart of videojs-contrib-hls - we faced a problem: How do we determine if the output from Mux.js is correct?
+
+Early on we managed to figure out how to coax FFmpeg into creating fragmented-MP4s from TS segments that would play back in a browser with media-source-extensions (at the time that meant only Chrome.) But we needed a simple way to compare the output of our transmuxer with what was produced by FFmpeg. The comparison had to be aware of the MP4 format since the two outputs are extremely unlikely to be byte-identical.
```

Comment:
I would write out `Media Source Extensions (MSE)` and just use `MSE` in the rest of the article.